index.bs

```diff
@@ -2421,6 +2415,106 @@ This is because the first 37 bytes of the signed data in a FIDO U2F authenticati
+
+    In this use case the authenticator is [=something you have=] which also verifies [=something you know=] and/or [=something you
+    are=], depending on the particular [=authenticator=]. This use case is only available with [=authenticators=] that support
+    both [=user verification=] and [=client-side-resident credential private keys=].
+
+
+### <dfn>Attachment Modality</dfn> ### {#sctn-authenticator-attachment-modality}
+
+[=Clients=] can communicate with [=authenticators=] using a variety of mechanisms. For example, a [=client=] MAY use a
+platform-specific API to communicate with an [=authenticator=] which is physically bound to a platform. On the other hand, a
+[=client=] can use a variety of standardized cross-platform transport protocols such as Bluetooth (see [[#transport]]) to discover
+and communicate with [=cross-platform attachment|cross-platform attached=] [=authenticators=]. We refer to [=authenticators=] that
+are part of the [=client=]'s platform as <dfn>platform authenticators</dfn>, while those that are reachable via cross-platform
+transport protocols are referred to as <dfn>roaming authenticators</dfn>.
+
+- A [=platform authenticator=] is attached using a platform-specific transport, and is usually not removable from the platform. A
+    [=public key credential=] bound to a [=platform authenticator=] is called a <dfn>platform credential</dfn>.
+
+- A [=roaming authenticator=] is attached using cross-platform transports. Authenticators of this class are removable from, and
+    can "roam" among, client platforms. A [=public key credential=] bound to a [=roaming authenticator=] is called a <dfn>roaming
+    credential</dfn>.
+
+Some [=platform authenticators=] could possibly also act as [=roaming authenticators=] depending on context. For example, a
+[=platform authenticator=] integrated into a mobile device could make itself available as a [=roaming authenticator=] via
+Bluetooth. In this case the [=client=] would identify it only as a [=roaming authenticator=], and not as a [=platform
+authenticator=].
+
+The primary use case for [=platform authenticators=] is to register a particular device as a "trusted device" available as a
+[=something you have=] [=authentication factor=] for future [=authentication=]. This gives the user the convenience benefit of not
+needing a [=roaming authenticator=] for future [=authentication ceremonies=], e.g., the user will not have to dig around in their
+pocket for their key fob or phone.
+
+The primary use case for [=roaming authenticators=] is for initial [=authentication=] on a new device, or on devices that are
+rarely used or do not include a [=platform authenticator=]; or when policy or preference dictates that the [=authenticator=] be
+kept separate from the [=clients=] it is used with. A [=roaming authenticator=] can also be used to hold backup [=public key
+credential|credentials=] in case another [=authenticator=] is lost.
+
```

Comment:
seems to me the above two use cases ought to be folded into the use cases proper. That's part of what I'm attempting to write up.

Reply:
Ok, you're welcome to open PRs into #956 if you come up with something.